### ISSUE TITLE
Requirements for installation are not clear

### ISSUE THREAD

**Issue by kjaquier** [seiyria/angular-bootstrap-slider#73]
> Just include `slider.js` and use the package `ui.bootstrap-slider`.

I actually had to include `seiyria-bootstrap-slider/dist/bootstrap-slider.min.js` and `seiyria-bootstrap-slider/dist/css/bootstrap-slider.min.css` too.

Also, looking at the code, I'm not sure it works without jQuery, like the original plugin is supposed to. It would really be a plus, since most angular modules for UI still require it...

**Comment by seiyria** [comment]
Yes, it is implied that bootstrap slider is already included. If you'd be willing to submit a PR on that, I'd take that.

You are correct, this does look for `$.fn.slider` and it is untested without jQuery. However, Angular, by default, has `jqLite` and IIRC if bootstrap-slider detects any jQuery namespace it will attach to it, which makes this a moot point. That said, if you have a plnkr that demonstrates this case, I'd be happy to take a look at it, but a PR would be overall a lot more helpful. As you said though, a lot of Angular plugins are just jQuery wrappers in disguise, so I'm not sure relying on jQuery is that much of a detriment.

**Comment by tobilg** [comment]
I second both points, it would be great if the actual installation hints would be complete. When using it without jQuery, I see an `$ is not defined` error in the console, so IMHO there's no automatic usage of `jqLite`.

**Comment by seiyria** [comment]
That's strange. In any event, if someone really needs the jQuery dependency to be removed, feel free to submit a PR. I suppose I'll flesh out the docs a bit more to say you do need to include bootstrap-slider (though I still feel this is a given).

**Comment by zauker** [comment]
Sorry guys, but i'm not understand what are the requirements.
For basic usage I need to install, bootstrap (and include the CSS), angular, angular-bootstrap-slider and?
seiyria/bootstrap-slider?
other package?
could you give me the minimun installation required via bower command?
I'm trying but i have not errors bat also i have not the slider too.
Thanks

**Comment by seiyria** [comment]
Yes, you need to install bootstrap-slider. That is implied when I say:

> See bootstrap-slider for examples and options.

and

> You will also need to include bootstrap-sliders CSS and JS.

bower should _already_ install `seiyria-bootstrap-slider` as well.

**Comment by zauker** [comment]
Thank you for your fast response,
I've made a little example on plunker but it doesn't works.

http://plnkr.co/edit/wumvDyTDAZiG4P2KlsLQ?p=preview

Coul'd you check if I miss something?
Thanks

**Comment by seiyria** [comment]
Yes, you forgot to require the module `ui.bootstrap-slider`.

**Comment by zauker** [comment]
Concerning _ui.bootstrap-slider_ you mean the Angular UI Bootstrap or jQuery ones?

http://angular-ui.github.io/bootstrap/ 
https://jquery-ui-bootstrap.github.io/jquery-ui-bootstrap/components.html#slider

Could you kindly write me the right ui.bootstrap-slider project page (or bower or CDN link) so i'll try to add into my sample.
thanks

**Comment by seiyria** [comment]
... You literally just have to add it as a module dependency, like so:

```js
(function () {
	var app = angular.module("sliderApp", ['ui.bootstrap-slider']);

	app.controller('sliderCtrl', function($scope, $http) {
		this.sliderValue = 24;
	});
})();
```

You can see [here](https://github.com/seiyria/angular-bootstrap-slider/blob/master/slider.js#L1) that that is what I've chosen to call this module.

**Comment by zauker** [comment]
Oh damn,
I was checking about the files inclusions and I did not realize that I forgot to load the module.
Now it works fine, thank you for your support.

**Comment by tanzaho** [comment]
This is pretty obvisous, but I guess, people might forget about the module inclusion at first (I did).
This might be great to add a line about it in the doc.
